### ISSUE TITLE
Use Prism 5.0

### ISSUE THREAD

**Issue by chandramouleswaran** [chandramouleswaran/Wide#62]
Update to Prism 5.0

http://msdn.microsoft.com/en-us/library/gg406140.aspx

http://www.microsoft.com/en-us/download/details.aspx?id=42537

**Comment by ghost** [comment]
lol, they changed 4.2 to 5.0. 
So i think Prism was a good choice ,if its actively developed.
keep it up! :)

Any plans for PopupWindowAction?

**Comment by chandramouleswaran** [comment]
Yes - I have not added 5.0 yet. Planning to add it over this weekend.

On Tue, Apr 22, 2014 at 9:18 AM, XBog notifications@github.com wrote:

> lol, they changed 4.2 to 5.0.
> So i think Prism was a good choice ,if its actively developed.
> keep it up! :)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/chandramouleswaran/Wide/issues/62#issuecomment-41044817
> .

## 

# 

Regards

# Chandramouleswaran
